rnd/autogpt_builder/src/lib/autogpt-server-api/types.ts

```diff
@@ -182,3 +183,10 @@ export type User = {
   id: string;
   email: string;
 };
+
+export enum BlockUIType {
+  STANDARD = "Standard",
+  INPUT = "Input",
+  OUTPUT = "Output",
+  NOTE = "Note",
+}
```

Comment:
`enum` makes code a bit more verbose and you assign string value to each anyway, so this would be better imo:
```suggestion
export type BlockUIType =
  | "Standard"
  | "Input"
  | "Output"
  | "Note"
```

Reply:
I tried it but I personally prefer the usability of enums in switches rather than just strings.